tests/contrib/grpc_aio/test_grpc_aio.py

```diff
@@ -222,11 +294,15 @@ async def test_pin_not_activated(server_target, tracer):
 
 
 @pytest.mark.asyncio
-async def test_pin_tags_put_in_span(patch_grpc_aio, tracer):
+@pytest.mark.parametrize(
+    "servicer",
+    [_SyncHelloServicer, _HelloServicer],
+)
+async def test_pin_tags_put_in_span(patch_grpc_aio, servicer, tracer):
```

Comment:
```suggestion
async def test_pin_tags_put_in_span(servicer, tracer):
```

Now that `autouse=True`, no need to pass `patch_grpc_aio`.

Reply:
Fixed in da5c6fa.